packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -4,12 +4,14 @@ import * as cxapi from '@aws-cdk/cx-api';
 import { Construct } from 'constructs';
 import { Connections } from './connections';
 import { CfnSecurityGroup, CfnSecurityGroupEgress, CfnSecurityGroupIngress } from './ec2.generated';
-import { IPeer } from './peer';
+import { IPeer, Peer } from './peer';
 import { Port } from './port';
 import { IVpc } from './vpc';
 
 const SECURITY_GROUP_SYMBOL = Symbol.for('@aws-cdk/iam.SecurityGroup');
 
+const SECURITY_GROUP_DISABLE_INLINE_RULES_CONTEXT_KEY = '@aws-cdk/aws-ec2.securityGroupNoInlineRules';
```

Comment:
I'm not set on this path - seems like something that should be defined by the project maintainers and not a random first-time contributor - you let me know and I'll change it.

Reply:
I think it makes most sense to align this with the `SecurityGroup` property that controls the same behavior. So that would make it:

```
@aws-cdk/aws-ec2.securityGroupDisableInlineRules
```